bin/foreman-proxy-certs-generate

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
```

Comment:
I thin this is a risky change to be making here. I think the intent is in a good place but this adds risk, testing burden, and potential for compounding issues. I would prefer split this off to be made after the merge as a re-factor that is independent.

Reply:
On the other hand, the way it's dealing with directories would already require significant changes. That made me consider this a lower risk.

Reply:
Can you explain what you mean by "dealing with directories" and why that would require significant change givne this all works as is today?